monkey/infection_monkey/master/ip_scan_results.py

```diff
@@ -0,0 +1,14 @@
+from dataclasses import dataclass
+from typing import Dict
+
+from infection_monkey.i_puppet import FingerprintData, PingScanData, PortScanData
+
+Port = int
+FingerprinterName = str
+
+
+@dataclass
+class IPScanResults:
+    ping_scan_data: PingScanData
+    port_scan_data: Dict[Port, PortScanData]
+    fingerprint_data: Dict[FingerprinterName, FingerprintData]
```

Comment:
Why is this a dataclass while `PortScanData`, `PingScanData` etc, are named tuples?

Reply:
The types were starting to get more complicated. For example, `PingScanData` consists of a bool and a string, while IPScanResults is storing other structures in dicts. I thought having the type hints would make it easier to understand what IPScanResults is.